index.js

```diff
@@ -172,3 +175,24 @@ allNavlinks.forEach((link) => {
     closeMenu();
   });
 });
+
+// FORM VALIDATE
+
+function errorMessage() {
+  if (email.value !== email.value.toLowerCase()) {
+    error.textContent = 'Please ensure that email address must be in lower case';
+  }
+}
+
+email.addEventListener('input', () => {
+  if (email.value === email.value.toLowerCase()) {
+    error.textContent = '';
+  }
+});
+
+form.addEventListener('submit', (event) => {
+  if (email.value !== email.value.toLowerCase()) {
+    event.preventDefault();
+    errorMessage();
+  }
+});
```

Comment:
- [OPTIONAL] Your email validation works great :+1: :clap: But following the best Javascript practices for writing dry code, There are few repeated codes that could basically be combined in one function, I'll suggest compressing your current implementation into something like this in the screenshot below, it makes your code DRY and SIMPLE. :+1: :smile: 

- - CODE SNIPPET:
![Screenshot from 2023-06-22 14-13-08](https://github.com/blessing-michael/portfolio/assets/107632780/e5d0cec0-b3ab-48c5-b497-27463c37f7b2)